packages/react/src/analytics-constants.doc.ts

```diff
@@ -0,0 +1,30 @@
+import {ReferenceEntityTemplateSchema} from '@shopify/generate-docs';
+
+const data: ReferenceEntityTemplateSchema = {
+  name: 'Analytics constants',
+  category: 'utilities',
+  isVisualComponent: false,
+  related: [],
```

Comment:
Do you want to add some links to this related area, to the other analytics docs?